src/System.Reflection/tests/PropertyInfo/PropertyInfo_propertyTests.cs

```diff
@@ -13,230 +13,88 @@ namespace System.Reflection.Tests
 {
     public class PropertyInfoPropertyTests
     {
-        //Verify CanRead for read write PropertyInfo
-        [Fact]
-        public static void TestCanRead1()
+        [Theory]
+        [InlineData("MyPropAA", true)]
+        [InlineData("MyPropBB", true)]
+        [InlineData("MyPropCC", false)]
+        public static void TestCanRead(String propName, bool boolean)
```

Comment:
Minor style point but if you're updating some of these tests, you could rename them. I don't see the value in the `Test` prefix. Also perhaps the param names can be more descriptive.
e.g. `public static void TestCanRead(String propName, bool boolean)` goes to `public static void CanRead(String propName, bool expected)`

I'm not sure if we have a preference for static vs non-static test methods, but that's tiny